project-reports/2024/2024-Q3-Hyperledger-Aries.md

```diff
@@ -0,0 +1,144 @@
+
+## Questions/Issues for the TOC
+
+None at this time.
+
+## Releases
+
+The following Aries releases occurred in the last two quarters (including the
+time covered by the Annual Report and so not yet reported):
+
+- [Aries Cloud Agent Python] -- 0.11.1, 0.12.0, 0.12.1
+- [Aries Askar] -- None
+- [Aries VCX] -- v0.64.0
+
+Interoperability status can be seen
+here: [https://aries-interop.info](https://aries-interop.info).
+
+## Overall Activity in the Past Quarter
+
+Per the [Aries Activity Dashboard] for the months of April-June 2024, Aries
+codebases had 3185 Commits and 275 contributors representing at least 28
+organizations.  The numbers do include the repositories
+that have moved from Hyperledger to the Open Wallet Foundation, as well as some
+obsolete (but unchanging) repositories. Per Ry Jones there is not a good way for
+us to control the included list of repos.
+
+[Aries Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=aries&routedFrom=Github&bestPractice=false&dateFilters=Last%20Quarter&dateRange=2024-04-01%20to%202024-06-30&compare=PP&granularity=week&hideBots=true
+
+- The [Indy SDK] has been removed from ACA-Py, eliminating the older component in favour of newer and more capable
+  replacements ([Aries Askar], [Indy VDR], and [AnonCreds RS]). Good to see that
+  transition!
+  - **NOTE: THIS DOES NOT REMOVE INDY SUPPORT FROM ACA-PY!**
+- Support for AnonCreds in W3C VCDM Format in Aries has been implemented in [Aries Cloud Agent Python].
+- Work has begun on adding [DIDComm v2] support in [ACA-Py].
+
+[Aries Askar]: https://github.com/hyperledger/aries-askar
+[Indy SDK]: https://github.com/hyperledger/indy-sdk
+[DIDComm v2]: https://identity.foundation/didcomm-messaging/spec/
+[ACA-Py]: https://aca-py.org
+
+## Aries Framework Reports
+
+### [Aries VCX]
+
+Lots of work has been happening on Aries VCX with the release of 0.64.0 and
+0.65.0, and it has also gained new maintainers, contributors and support from
+Instnt.
+
+The releases have included updates to the AATH (Aries Agent Test Harness) Aries
+VCX Backchannel to include the latest 0.65.0 updates and implement processes for
+continuous deployment of the backchannel. As part of the efforts updating the
+backchannel, work has occurred to resolve interoperability issues between Aries
+VCX and Aries Cloud Agent Python (ACA-Py) within DID Exchange, adding support
+for DID Exchange 1.1 and did:peer numalgo 4.
+
+Release 0.65.0 marks the planned final release including support for the
+indy-sdk, credx, and indyvdrtools crates and features in Aries VCX. The
+replacement for these crates are already implemented (Aries Askar, anoncreds-rs,
+and indy-vdr) in Aries VCX. Additional info or discussion can be found here.
+Work is underway to remove these dependencies from the project.
+
+Work is in-progress on fixing and enhancing fixing Android support, as well as
```

Comment:
Would be good to have at the top of this section what is VCX and how it's different from the Bifold sub-project (my understanding is both are client side SDK targeted at mobile app development?).